Add 10.10 release notes (#87)
mondoohq/docs: releases/2024-04-02-mondoo-10.10-is-out.md

---
slug: mondoo-10.10-is-out/
title: Mondoo 10.10 is out!
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
tags: [release, mondoo]
---

> ## 🥳 Mondoo 10.10 is out! This release includes xz vulnerability detection, expanded AWS asset inventory, and more!

Reviewer: ```suggestion
## 🥳 Mondoo 10.10 is out! This release includes XZ Utils vulnerability detection, expanded AWS asset inventory, and more!
```

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

> ### xz vulnerability policy

Reviewer: ```suggestion
### XZ Utils vulnerability policy
```

The recent xz supply chain attack in xz 5.6.0 and 5.6.1 (CVE-2024–3094) thankfully didn't make it in any mainstream enterprise Linux distributions. There's still a significant risk if employees are running rolling distributions or pre-releases of upcoming Linux distros. To quickly evaluate your CVE-2024–3094 expose we've created a new xz Vulnerability (CVE-2024–3094) policy that looks for xz 5.6.0/5.6.1 on impacted Linux releases:

- Alpine
- Arch
- Debian trixie/sid
- Fedora 40
- Kali 2024.1
- openSUSE Tumbleweed

> ![xz Policy affected assets](/img/releases/2024-04-02-mondoo-10.10-is-out/xz_policy.png)

Reviewer: ```suggestion
![XZ Vulnerability Policy affected assets](/img/releases/2024-04-02-mondoo-10.10-is-out/xz_policy.png)
```

## 🧹 IMPROVEMENTS

### Improved AWS asset overview information

> Get the context you need to resolve security findings quickly with expanded asset overview information on AWS assets:

Reviewer: ```suggestion
Get the context you need to resolve security findings quickly with expanded overview information on AWS assets:
```

- Volume size on EBS volumes and snapshots
- Database engine version on RDS instances
- Storage size and type on RDS instances
- Table size on DynamoDB tables
- Retention time on CloudWatch log groups

![RDS instance with expanded asset overview](/img/releases/2024-04-02-mondoo-10.10-is-out/asset_overview.png)

### Expanded Endpoint Detection and Response policy support

Detect the ESET EDR in the Endpoint Detection and Response (EDR) policy.

### New Terraform checks in CIS GCP Foundation policy

> Flag critical security misconfigurations before they're ever running in your infrastructure with expanded Terraform config checks in the CIS Google Cloud Platform Foundation policy. New checks evaluate Terraform configs for proper [GCP uniform bucket level access](https://cloud.google.com/storage/docs/uniform-bucket-level-access) setup.

Reviewer: ```suggestion
Flag critical security misconfigurations before they ever run in your infrastructure with expanded Terraform config checks in the CIS Google Cloud Platform Foundation policy. New checks evaluate Terraform configs for proper [GCP uniform bucket level access](https://cloud.google.com/storage/docs/uniform-bucket-level-access) setup.
```

### Fedora 40 EOL/CVE detection

> The [Fedora 40 beta](https://www.redhat.com/en/blog/fedora-40-beta-now-available) is now available for testing, and Mondoo is ready with CVE and EOL detection for this upcoming Linux release. Keep your test systems safe from critical vulnerabilities such as the compromised xz release (CVE-2024–3094) that originally shipped in this beta.

Reviewer: ```suggestion
The [Fedora 40 beta](https://www.redhat.com/en/blog/fedora-40-beta-now-available) is now available for testing, and Mondoo is ready with CVE and EOL detection for this upcoming Linux release. Keep your test systems safe from critical vulnerabilities such as the compromised XZ release (CVE-2024–3094) that originally shipped in this beta.
```

### Resource improvements

#### aws.autoscaling.groups

- Improve resource default values
- New `availabilityZones` field
- New `capacityRebalance` field
- New `defaultInstanceWarmup` field
- New `desiredCapacity` field
- New `instances` field
- New `maxInstanceLifetime` field

#### aws.cloudfront.distributions

- New `cnames` field

## 🐛 BUG FIXES AND UPDATES

- Improve performance of AWS cloud detection.

> - Fix Windows policies with multi-language support to rely on the system language not the locale.

Reviewer: ```suggestion
- Fix Windows policies with multi-language support to rely on the system language instead of the locale.
```

- Simplify the Linux server installation instructions.
- Support vulnerability scanning of RPMs with a `^` symbol in the name.
- Update additional CIS GCP Foundations checks to work against Terraform configs.

> - Fix `CIS VMware ESXi 6.7 Benchmark - Corporate/Enterprise Environment` policy to only apply to VMware 6.

Reviewer: ```suggestion
- Fix the `CIS VMware ESXi 6.7 Benchmark - Corporate/Enterprise Environment` policy to only apply to VMware 6.
```
